src/components/primitives/Readme.md

```diff
@@ -0,0 +1,9 @@
+Primitives are building blocks of your application. Since primitives can be used anywhere in the UI they should be agnostic of their environment.
+
+This means primitives:
```

Comment:
Also:

- should not define width and height;
- should not expect to be inside a Flexbox or absolutely positioned container;

Reply:
Good one. Propose a change?